presentation/components/src/presentation-components/table/DataProvider.ts

```diff
@@ -193,8 +193,10 @@ export class PresentationTableDataProvider extends ContentDataProvider implement
         direction: (this._sortDirection === UiSortDirection.Descending) ? SortDirection.Descending : SortDirection.Ascending,
       };
     }
-    if (this._filterExpression)
-      overrides.filterExpression = this._filterExpression;
+    if (this._filterExpression) {
+      overrides.filterExpression = this._filterExpression; // eslint-disable-line deprecation/deprecation
```

Comment:
Why set both instead of just the new one?

Reply:
Thought that there might be a case when newer frontend calls older backend that does not support the new `fieldsFilterExpression`. If that's not a case, I can remove `filterExpression`

Reply:
Bumped PresentationRPCInterface version and left only the new `fieldsFilterExpression` property when creating `DescriptorOverrides`